### PR TITLE
Rspec test

### PR DESCRIPTION
**Pull request by AshakaE**
- Created tests for board class and player class

### CODE REVIEW

File: spec/board_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      it { is_expected.to be_truthy }
+    end
+  end
+
+  describe '#board_full?' do
+    context 'to check if the board is full' do
+      it { is_expected.not_to be_nil }
+    end
+  end
+end
```
Comment (lekegitrepo) [inline spec/board_spec.rb]: - **_These specs do not really test the board object. For example, if anything is broken in the board module( I changed the @size value to 0 and 100) all the tests still pass. Improve the specs to test the object as you did with the `player_spec`. And remember to cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @AshakaE and @KumarAmitt :clap: :clap:
#### You have done a good job so far :muscle: I checked the overall project and I can see that you did a good job building the `TicTacToe` game. However, you still need to improve some of the requirements you implemented while writing the tests. Check my comments below :point_down:

#### 1) README:
 - Add instruction on how to run the tests in the readme.

#### 2) RUBOCOP ERRORS:
 - Try to fix the rubocop errors.

#### 3) ADD MORE TESTS:
 - All the public methods of the classes implemented in the project should be tested ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/all_tic_tac_toe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)). Add tests for the `utils` module.

#### _Reviewed by Temitayo Adeleke_

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑 🔧 
### Hi there Team ✋🏽 
Nice work with the RSpec testing. 

- [x] You just have to add instructions for testing in your Readme file for this project to be approved.

_Feel free to reach out to me on Slack if you have any questions, Witah Georjane_ 🤝 
Good luck on your project and Happy Coding 💻 

**Witah Georjane
Code Reviewer at Microverse Inc
[portfolio](https://witahgeorjane.tech/) | [linkedIn](https://www.linkedin.com/in/witah-georjane/)**

**Review by cristianCeamatu — APPROVED** [review]
### 🥇 APPROVED 🥇 Congratulations and good luck with the next projects!

If you ever need any help or notice that you are wasting too many hours on a bug don't hesitate to contact me on Slack.

Best regards,
Cristian Ceamatu
Slack: Cristian Viorel Ceamatu

**Pull request merged**